bin/integrate_sce.R

```diff
@@ -131,5 +131,13 @@ integrated_sce <- integrated_sce |>
   scater::runUMAP(dimred = glue::glue("{opt$method}_PCA"),
                   name = glue::glue("{opt$method}_UMAP"))
 
+# add method to integrated sce
+existing_methods <- metadata(integrated_sce)$integration_methods
+if(!is.null(existing_methods)){
+  metadata(integrated_sce)$integration_methods <- c(existing_methods, opt$method)
+} else {
+  metadata(integrated_sce)$integration_methods <- opt$method
+}
```

Comment:
I'm pretty sure that `c(NULL, something)` is equivalent to  `c(something)`, so I think you can reduce this to: 

```suggestion
existing_methods <- metadata(integrated_sce)$integration_methods
metadata(integrated_sce)$integration_methods <- c(existing_methods, opt$method)
```